### PR TITLE
Add some clarification to Fronius integration docs

### PR DESCRIPTION
**Pull request by nagyrobi**
## Proposed change

Fix some typos + add information on how to find out the device IDs from the API, which is required prior to setting up the integration.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fronius.markdown [hunk 1]
```diff
@@ -13,7 +13,7 @@ ha_platforms:
   - sensor
 ---
 
-The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors in Home Assistant.
+The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and presents the values as sensors in Home Assistant. Data is gathered directly through the system's datamanager card connected to your local network, this integration doesn't access the cloud. You will need to either set a static IP or assign a static DHCP lease, or alternatively access it through the local DNS name if your network is properly configured.
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: This is now an introduction and an instruction in one. Please split the prerequisites from the introduction.

File: source/_integrations/fronius.markdown [hunk 2]
```diff
@@ -22,14 +22,14 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
...
-    resource: FRONIUS_URL
+    resource: FRONIUS_LOCAL_API_URL
     monitored_conditions:
     - sensor_type: inverter
 ```
 
 {% configuration %}
 resource:
-  description: "The URL of the Fronius device (e.g., `http://192.0.2.0` or `http://fronius.local`)"
+  description: "The URL of the API on the Fronius datamanager card (e.g., `http://192.0.2.0` or `http://fronius.local`)"
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: Is the API different from regular device access? As in, as it specifically mentions it, is that needed?

File: source/_integrations/fronius.markdown [hunk 3]
```diff
@@ -90,6 +90,19 @@ there might be no sensor providing photovoltaic related data.
 This does not need to be problematic as the values will be added on sunrise,
 when the Fronius devices begins providing the needed data.
 
+## Finding out devices IDs
+
+To find out the device ID of the inverter visit the URL:
+http://your_datamanager_card_ip/solar_api/v1/GetPowerFlowRealtimeData.fcgi
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: Don't use bare URLs, use all caps for replacement values.

File: source/_integrations/fronius.markdown [hunk 4]
```diff
@@ -90,6 +90,19 @@ there might be no sensor providing photovoltaic related data.
...
+In the returned json, under the key Body > Data > Inverters you should see your inverters listed with IDs starting from 1.
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
In the returned JSON, under the key Body > Data > Inverters you should see your inverters listed with IDs starting from 1.
```

File: source/_integrations/fronius.markdown [hunk 5]
```diff
@@ -90,6 +90,19 @@ there might be no sensor providing photovoltaic related data.
...
+
+To find out the device ID of the meter visit the URL:
+http://your_datamanager_card_ip/solar_api/v1/GetMeterRealtimeData.cgi?Scope=System
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: Don't use bare URLs, use all caps for replacement values.

File: source/_integrations/fronius.markdown [hunk 6]
```diff
@@ -90,6 +90,19 @@ there might be no sensor providing photovoltaic related data.
...
+In the returned json, under the key Body > Data you should see your meters listed with IDs starting from 0.
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
In the returned JSON, under the key Body > Data you should see your meters listed with IDs starting from 0.
```

File: source/_integrations/fronius.markdown [hunk 7]
```diff
@@ -90,6 +90,19 @@ there might be no sensor providing photovoltaic related data.
...
+
+To find out which API version your system runs, visit the URL:
+http://your_datamanager_card_ip/solar_api/GetAPIVersion.cgi
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: Don't use bare URLs, use all caps for replacement values

File: source/_integrations/fronius.markdown [hunk 8]
```diff
@@ -108,3 +121,7 @@ sensor:
       device: 0
     - sensor_type: power_flow
 ```
+
+## Note
+
+Fronius often provides firmware updates for the datamanager interfaces and the devices in their system, it's recommended to check and apply them regurarly.
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: How is this related to the integration?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fronius.markdown (hunk 1 above)
Comment (nagyrobi) [reply source/_integrations/fronius.markdown]: Done, thanks for the observation.

File: source/_integrations/fronius.markdown (hunk 2 above)
Comment (nagyrobi) [reply source/_integrations/fronius.markdown]: Clarified that only root URL is needed.

File: source/_integrations/fronius.markdown (hunk 3 above)
Comment (nagyrobi) [reply source/_integrations/fronius.markdown]: Done.

File: source/_integrations/fronius.markdown (hunk 5 above)
Comment (nagyrobi) [reply source/_integrations/fronius.markdown]: Done.

File: source/_integrations/fronius.markdown (hunk 7 above)
Comment (nagyrobi) [reply source/_integrations/fronius.markdown]: Done.

File: source/_integrations/fronius.markdown (hunk 8 above)
Comment (nagyrobi) [reply source/_integrations/fronius.markdown]: It is in the aspect that some parts of the API used by this integration are missing in older firmware revisions of the datamanager card. I thought it could be useful to point people in the direction of updating it to the lastest before starting to look for issues in HA.
Clarified the statement accordingly.

**Comment by nagyrobi** [comment]
recheck

File: source/_integrations/fronius.markdown [hunk 9]
```diff
@@ -13,7 +13,11 @@ ha_platforms:
   - sensor
 ---
 
-The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors in Home Assistant.
+The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and presents the values as sensors in Home Assistant. Data is gathered directly through the system's datamanager card connected to your local network, this integration doesn't access the cloud.
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and presents the values as sensors in Home Assistant. Data is gathered directly from a Fronius device connected to your local network, this integration doesn't access the cloud. 
```

File: source/_integrations/fronius.markdown [hunk 10]
```diff
@@ -13,7 +13,11 @@ ha_platforms:
   - sensor
 ---
 
-The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors in Home Assistant.
+The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and presents the values as sensors in Home Assistant. Data is gathered directly through the system's datamanager card connected to your local network, this integration doesn't access the cloud. 
+
+## Prerequisites
+
+You will need to either set a static IP on the datamanager card or assign a static DHCP lease for it, or alternatively access it through the local DNS name if your network is properly configured for this.
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
You will need to either set a static IP on the Fronius device or assign a static DHCP lease for it, or alternatively access it through the local DNS name if your network is properly configured for this.
```

File: source/_integrations/fronius.markdown [hunk 11]
```diff
@@ -22,14 +26,14 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
 ```yaml
 sensor:
   - platform: fronius
-    resource: FRONIUS_URL
+    resource: FRONIUS_CARD_LOCAL_URL
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
    resource: FRONIUS_URL_OR_IP
```

File: source/_integrations/fronius.markdown [hunk 12]
```diff
@@ -22,14 +26,14 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
...
     monitored_conditions:
     - sensor_type: inverter
 ```
 
 {% configuration %}
 resource:
-  description: "The URL of the Fronius device (e.g., `http://192.0.2.0` or `http://fronius.local`)"
+  description: "The root URL of the Fronius datamanager card (e.g., `http://192.0.2.0` or `http://fronius.local`)"
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
  description: "The hostname or IP address of the Fronius device (e.g., `192.0.2.0` or `http://fronius.local`)"
```

File: source/_integrations/fronius.markdown [hunk 13]
```diff
@@ -90,6 +94,19 @@ there might be no sensor providing photovoltaic related data.
 This does not need to be problematic as the values will be added on sunrise,
 when the Fronius devices begins providing the needed data.
 
+## Finding out devices IDs
+
+To find out the device ID of the inverter visit the URL:
+http://FRONIUS_DATAMANAGER_CARD_IP/solar_api/v1/GetPowerFlowRealtimeData.fcgi
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
http://FRONIUS_IP/solar_api/v1/GetPowerFlowRealtimeData.fcgi
```

File: source/_integrations/fronius.markdown [hunk 14]
```diff
@@ -90,6 +94,19 @@ there might be no sensor providing photovoltaic related data.
...
+In the returned JSON, under the key Body > Data > Inverters you should see your inverters listed with IDs starting from 1.
+
+To find out the device ID of the meter visit the URL:
+http://FRONIUS_DATAMANAGER_CARD_IP/solar_api/v1/GetMeterRealtimeData.cgi?Scope=System
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
http://FRONIUS_IP/solar_api/v1/GetMeterRealtimeData.cgi?Scope=System
```

File: source/_integrations/fronius.markdown [hunk 15]
```diff
@@ -90,6 +94,19 @@ there might be no sensor providing photovoltaic related data.
...
+In the returned JSON, under the key Body > Data you should see your meters listed with IDs starting from 0.
+
+To find out which API version your system runs, visit the URL:
+http://FRONIUS_DATAMANAGER_CARD_IP/solar_api/GetAPIVersion.cgi
```
Comment (farmio) [inline source/_integrations/fronius.markdown]:
```suggestion
http://FRONIUS_IP/solar_api/GetAPIVersion.cgi
```

**Review by farmio — COMMENTED** [review]
Gen24 devices don't have Datalogger / Datamanager cards anymore.

**Comment by nagyrobi** [comment]
Recheck

File: source/_integrations/fronius.markdown [hunk 16]
```diff
@@ -90,6 +94,19 @@ there might be no sensor providing photovoltaic related data.
 This does not need to be problematic as the values will be added on sunrise,
 when the Fronius devices begins providing the needed data.
 
+## Finding out devices IDs
+
+To find out the device ID of the inverter visit the URL:
+http://FRONIUS_IP/solar_api/v1/GetPowerFlowRealtimeData.fcgi
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
`http://FRONIUS_IP/solar_api/v1/GetPowerFlowRealtimeData.fcgi`
```

File: source/_integrations/fronius.markdown [hunk 17]
```diff
@@ -90,6 +94,19 @@ there might be no sensor providing photovoltaic related data.
...
+In the returned JSON, under the key Body > Data > Inverters you should see your inverters listed with IDs starting from 1.
+
+To find out the device ID of the meter visit the URL:
+http://FRONIUS_IP/solar_api/v1/GetMeterRealtimeData.cgi?Scope=System
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
`http://FRONIUS_IP/solar_api/v1/GetMeterRealtimeData.cgi?Scope=System`
```

File: source/_integrations/fronius.markdown [hunk 18]
```diff
@@ -90,6 +94,19 @@ there might be no sensor providing photovoltaic related data.
...
+In the returned JSON, under the key Body > Data you should see your meters listed with IDs starting from 0.
+
+To find out which API version your system runs, visit the URL:
+http://FRONIUS_IP/solar_api/GetAPIVersion.cgi
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
`http://FRONIUS_IP/solar_api/GetAPIVersion.cgi`
```

File: source/_integrations/fronius.markdown [hunk 19]
```diff
@@ -108,3 +125,7 @@ sensor:
       device: 0
     - sensor_type: power_flow
 ```
+
+## Note
+
+Fronius often provides firmware updates for the datamanager interfaces and the devices in their system, it's recommended to check and apply them regurarly. This integration relies on functionality present in rather recent firmware.
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
Fronius often provides firmware updates for the datamanager interfaces and the devices in their system, it's recommended to check and apply them regularly. This integration relies on functionality present in rather recent firmware.
```

**Review by frenck — APPROVED** [review]
Thanks, @nagyrobi 👍
